src/resolution.js

```diff
@@ -70,6 +81,25 @@ var ri = module.exports = {
 	},
 
 	/**
+	* Update the common measured boundary object. This object is used as "what size screen are we
+	* looking at". Providing no arguments has no effect and updates nothing.
+	*
+	* @memberOf ui/resolution
```

Comment:
Can you update this module to match the Enyo style

Reply:
Just remove this line, right? I had removed it in a previous commit and I had to redo this a few times, and I must have forgotten about this in the latest attempt.